nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -857,6 +859,9 @@ public void onTrigger(ProcessContext context, ProcessSession session) throws Pro
                 statusAttributes.put(STATUS_MESSAGE, statusMessage);
                 statusAttributes.put(REQUEST_URL, url.toExternalForm());
                 statusAttributes.put(TRANSACTION_ID, txId.toString());
+                if(context.getProperty(PROP_FOLLOW_REDIRECTS).asBoolean()) {
```

Comment:
Rather than making the response.url attribute conditional on the Follow Redirects property, it seems like it would be useful to always add the attribute.  What do you think?

Reply:
Thank you for your feedback.  I debated the same thing but went for it this way to avoid duplication since if redirects are disabled, the "response" url would always be identical to the "request" url.  I'd be happy to always add it as well.  It might make some flows cleaner if they can rely on the "response" attribute always being set.  Let me submit that change.

Reply:
Thanks for the response and updates @pkelly-nifi, the changes look good.  Could you also update the InvokeHTTPTest to check for the existence of the Response URL attribute?  The `assertStatusCodeEquals()` method has a check for the InvokeHTTP.REQUEST_URL attribute, so just adding a line to check for the Response URL attribute should be sufficient.

Reply:
Just added the test.  Thank you for your help.